openingd/openingd.c

```diff
@@ -974,6 +974,14 @@ static u8 *fundee_channel(struct state *state, const u8 *open_channel_msg)
 	if (!state->upfront_shutdown_script[LOCAL])
 		state->upfront_shutdown_script[LOCAL] = no_upfront_shutdown_script(state, state);
 
+	/* Just before we accept, we tell the master that we are now
+	 * going to be a fundee, so it should fail any pending
+	 * `fundchannel_start` on this peer, and fail all future ones
+	 * as well.
+	 */
+	msg = towire_openingd_will_be_fundee(NULL);
```

Comment:
You don't need a new wire for this, see `opening_got_offer` in `lightningd/opening_control.c`.